### PR TITLE
Fix Implements part of object docs.

### PR DESCRIPTION
**Pull request by EPashkin**
Part of https://github.com/gtk-rs/gir/issues/500

cc @GuillaumeGomez, @sdroege

### CODE REVIEW

File: gio/vendor.md [hunk 1]
```diff
@@ -1178,11 +1178,11 @@ application is not expected to manually register itself and check
 if it is the primary instance. Instead, the `main` function of a
 `Application` should do very little more than instantiating the
 application instance, possibly connecting signal handlers, then
-calling `ApplicationExt::run`. All checks for uniqueness are done
+calling `Application::run`. All checks for uniqueness are done
```
Comment (EPashkin) [inline gio/vendor.md:1181]: This happened because `Application::run` now ignored and actually `ApplicationExtManual::run`.

**Comment by EPashkin** [comment]
Please check how look Implements section for Gtk::Application

File: gtk/vendor.md [hunk 2]
```diff
@@ -1654,7 +1654,7 @@ session while inhibitors are present.
 
 # Implements
 
-[`ApplicationExt`](trait.ApplicationExt.html), [`ApplicationExt`](trait.ApplicationExt.html), [`ObjectExt`](trait.ObjectExt.html), [`ActionGroupExt`](trait.ActionGroupExt.html), [`ActionMapExt`](trait.ActionMapExt.html)
+[`GtkApplicationExt`](trait.GtkApplicationExt.html), [`gio::ApplicationExt`](../gio/trait.ApplicationExt.html), [`glib::object::ObjectExt`](../glib/object/trait.ObjectExt.html), [`gio::ActionGroupExt`](../gio/trait.ActionGroupExt.html), [`gio::ActionMapExt`](../gio/trait.ActionMapExt.html)
```
Comment (EPashkin) [inline gtk/vendor.md:1657]: Now it looks like this

**Comment by EPashkin** [comment]
@GuillaumeGomez It look good can you merge this before gir (as it not contains any links),
and update site docs?
I can update package version here if you want.

**Comment by EPashkin** [comment]
*If it looks good ...

**Comment by GuillaumeGomez** [comment]
Thanks a lot!

**Pull request merged**